gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -194,7 +189,11 @@ fn v1_routes(gateway: Arc<Gateway>, task_group: TaskGroup) -> Router {
         .route(CONNECT_FED_ENDPOINT, post(connect_fed))
         .route(LEAVE_FED_ENDPOINT, post(leave_fed))
         .route(BACKUP_ENDPOINT, post(backup))
-        .route(PAY_INVOICE_SELF_ENDPOINT, post(pay_invoice_self))
+        .route(
+            CREATE_BOLT11_INVOICE_FOR_OPERATOR_ENDPOINT,
+            post(create_invoice_for_self),
```

Comment:
Why does this need to be authenticated? Do we care if "others" create invoices and send money to the gateway?

Reply:
As its for the operator to manage liquidity seems like this should be gated.

Reply:
But why? Other users might want to generate an invoice for paying the gateway too, doesn't seem like we should prevent that?

Reply:
In what situation you want users to just randomly send you money? Furthermore, if its not public we can return an admin error here.